apps/native/app/(tabs)/wallet/[walletId].tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useMemo, useState } from "react";
+import {
+    SafeAreaView,
+    ScrollView,
+    StyleSheet,
+    Text,
+    TouchableOpacity,
+    View,
+} from "react-native";
+import { Stack, useLocalSearchParams } from "expo-router";
+import { WalletActivity } from "../../../components/wallet/WalletActivity";
+import { WalletDetails } from "../../../components/wallet/WalletDetails";
+import { Colors } from "../../../constants/Colors";
```

Comment:
Let’s switch from ../../../ to aliased imports (@/...), cleaner, maintainable, and future-proof.